SpriterDotNet/Parser/XmlSpriterParser.cs

```diff
@@ -24,7 +26,14 @@ public Spriter Parse(string data)
 
         public bool CanParse(string data)
         {
-            return data.StartsWith(XmlStart);
+            return data != null && data.StartsWith(XmlStart);
```

Comment:
I understand what you're saying and I think that's a good approach in certain situations. However, in this instance I believe it's better to assume the input string is completely valid. Otherwise, we should also be trimming the data because StartsWith wouldn't return the expected result if the string has spaces in the beginning. Also, we should check if the string contains valid xml, etc. If we go this way, we'd have a lot of logic in every parser and that would induce a lot of clutter. When writing library code, I prefer delegating that responsibility to the caller.